src/lib/support/BUILD.gn

```diff
@@ -211,6 +235,7 @@ static_library("support") {
     "ThreadOperationalDataset.h",
     "TimeUtils.cpp",
     "TimeUtils.h",
+    "UnitTestContext.h",
```

Comment:
Should this go in the main /support since it is for unit tests?

Reply:
I think some moving around should be done ... however for now I just report what we have today. Will take a look at how painful this is to be done in this PR - if easy I will make the change, otherwise I will have a followup.

Reply:
moved it to `testing_nlunit`.